controllers/kubeadmconfig_controller_test.go

```diff
@@ -56,6 +56,45 @@ func setupScheme() *runtime.Scheme {
 	return scheme
 }
 
```

Comment:
Do you want to add one line here what is this test about? For example What should it test and what is expected result?

Reply:
Done